src/components/QuestionPlay/index.tsx

```diff
@@ -79,15 +79,27 @@ export default function QuestionPlay({
             Question {selectedQuestion.questionNum}
           </Title>
           {negativePointsMultiplier === 0 ? (
-            <Badge color={getBadgeColor(points, minQuestionPoint, maxQuestionPoint)} variant="filled">
+            <Badge
+              styles={{
```

Comment:
rename the function as `getBadgeStyles` and return whole style object
```
{
                root: {
                  backgroundColor: getPointsColor(points, minQuestionPoint, maxQuestionPoint).color,
                },
              }
```

Reply:
This function is used in other places where I am using the backgroundColor returned from the function.